numpy/linalg/umath_linalg.c.src

```diff
@@ -706,6 +706,11 @@ dump_@TYPE@_matrix(const char* name,
  *****************************************************************************
  */
 
+#ifndef min
+#define min(x,y) ((x<y)?x:y)
```

Comment:
Wouldn't "right" look like an inline function then, to make `max(a++, b++)` work correctly?